src/dependency.ts

```diff
@@ -0,0 +1,88 @@
+/* eslint-disable max-classes-per-file */
+
+import { ReplaceSource } from "webpack-sources";
+import * as docGen from "react-docgen-typescript";
+
+// eslint-disable-next-line
+// @ts-ignore: What's the right way to refer to this one?
+import ModuleDependency from "webpack/lib/dependencies/ModuleDependency.js";
```

Comment:
All of them are exposed with typings if you import them from the root package:

``` js
import webpack from "webpack";

const {
  Dependency,
  dependencies: { ModuleDependency },
  util: { serialization: { makeSerializable } }
} = webpack;
```

Reply:
For some reason `makeSerializable` doesn't seem to be available at the type yet (I checked the whole file) and importing `NullDependency` leads to

```
'extends' clause of exported class 'DocGenDependency' has or is using private name 'NullDependency'
```

I wonder if that's a type export issue as well.